prow/jobs/jobs.go

```diff
@@ -63,26 +63,33 @@ func (ps Presubmit) RunsAgainstBranch(branch string) bool {
 	return false
 }
 
+// Postsubmit runs on a timer.
+type Postsubmit struct {
+	Name string        `json:"name"`
+	Spec *kube.PodSpec `json:"spec,omitempty"`
+}
+
 type JobAgent struct {
```

Comment:
The duplication here feels weird and awkward. Why can't these job specs be unified?